openapi_gitbook.yaml

```
@@ -1321,7 +1329,13 @@ components:
 
         Characters included in the `context` parameter will not be counted toward billing.
       type: string
-      example: This is context. 
+      example: This is context.
+    ShowBilledCharacters:
+      description: |-
+        When true, the response will include an additional key-value pair with the key billed_characters and a 
+        value that is an integer showing the number of characters from the request that will be counted by DeepL 
```

Seems overly complex.
```suggestion
        When true, the response will include the billed_characters parameter, giving
        the number of characters from the request that will be counted by DeepL 
```